content/webapp/pages/homepage.tsx

```diff
@@ -197,7 +197,7 @@ const Homepage: FunctionComponent<Props> = ({
               className={font('wb', 1)}
             >
               <Space v={{ size: 'm', properties: ['margin-bottom'] }}>
-                <h1 className="no-margin">{homepageHeading}</h1>
```

Comment:
I had a play with this; the `no-margin` class isn't doing anything for us here – if you remove it, the homepage doesn't look any different (because of all the other spacing/margin elements in play).